File: app/models/referee_answer.rb
# == Schema Information
#
# Table name: referee_answers
#
#  id              :bigint(8)        not null, primary key
#  created_at      :datetime         not null
#  updated_at      :datetime         not null
#  answer_id       :bigint(8)        not null
#  question_id     :bigint(8)        not null
#  referee_id      :bigint(8)        not null
#  test_attempt_id :bigint(8)        not null
#  test_id         :bigint(8)        not null
#
# Indexes
#
#  index_referee_answers_on_answer_id        (answer_id)
#  index_referee_answers_on_question_id      (question_id)
#  index_referee_answers_on_referee_id       (referee_id)
#  index_referee_answers_on_test_attempt_id  (test_attempt_id)
#  index_referee_answers_on_test_id          (test_id)
#

class RefereeAnswer < ApplicationRecord
  belongs_to :answer
  belongs_to :question
  belongs_to :test
  belongs_to :referee, class_name: 'User'
  belongs_to :test_attempt

  delegate :correct?, to: :answer
end

File: dotnet/ManagementHub.Models/RefereeAnswer.cs
using System;
using System.Collections.Generic;

namespace ManagementHub.Models
{
    public partial class RefereeAnswer
    {
        public long Id { get; set; }
        public long RefereeId { get; set; }
        public long TestId { get; set; }
        public long QuestionId { get; set; }
        public long AnswerId { get; set; }
        public long TestAttemptId { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

        public virtual Answer Answer { get; set; } = null!;
        public virtual Question Question { get; set; } = null!;
        public virtual User Referee { get; set; } = null!;
        public virtual Test Test { get; set; } = null!;
        public virtual TestAttempt TestAttempt { get; set; } = null!;
    }
}
